kube/resources/airbyte-kill-stuck-pods/airbyte_kill_stuck_pods.js

```diff
@@ -0,0 +1,127 @@
+/**
```

Comment:
nit: do we still need this script? looks like you add it as a `ConfigMap` block in the .yaml file.

Reply:
I left it for convenience because operating on the block of data is hard. But I guess we can simply copy it out in a standalone file when and if we need to.